packages/realm/src/Configuration.ts

```diff
@@ -47,6 +50,15 @@ export type BaseConfiguration = {
   disableFormatUpgrade?: boolean;
   encryptionKey?: ArrayBuffer | ArrayBufferView | Int8Array;
   onMigration?: MigrationCallback;
+  /**
+   * A flag to indicate whether Realm should resolve
+   * embedded object constraints after migration. If this is `true` then all embedded objects
+   * without a parent will be deleted and every embedded object with multiple references to it
+   * will be duplicated so that every referencing object will hold its own copy of the
+   * embedded object.
+   * @default false
+   */
+  automaticallyHandleBacklinksInMigration?: boolean;
```

Comment:
Wondering if it'd be better to name this more in line with how you're describing it in the docs. Perhaps:
* `autoResolveEmbeddedConstraintsInMigration`
* `autoHandleEmbeddedConstraintsInMigration`
* (ps. Core uses the plural of Migration.)

Thoughts?

Reply:
Sounds good to me 👍